sql_bench/README.md

```diff
@@ -0,0 +1,31 @@
+# Local e2e SQL Benchmarks
+
```

Comment:
Can put the motivation in the pr description into the README.
e.g., it's most useful for ad-hoc benchmarking for developing optimizations

Reply:
True, usually in early stages we need to iterate fast. Benchmarking in local env makes sense.